lib/active_record/connection_adapters/spanner/schema_dumper.rb

```diff
@@ -27,6 +27,8 @@ def prepare_column_options column
             spec = { type: schema_type(column).inspect }.merge! spec
           end
 
+          spec[:array] = true if column.sql_type.start_with?('ARRAY<')
```

Comment:
feels like there's probably a safer way to determine if a `ActiveRecord::ConnectionAdapters::Spanner::Column` is `ARRAY` type, deferring to reviewers here

Reply:
I had a look at what is available here, and there isn't really anything else that you could use without changing the underlying fetching of column metadata. That would make this change much bigger, and I don't think there's any real problem with just checking the type name like this.